bayesian/observability/metrics.py

```diff
@@ -0,0 +1,83 @@
+"""Basic Metrics Collection logic."""
+
+import time
+
+from flask import current_app, g, request
+from prometheus_client import Counter, Histogram
+from prometheus_client.metrics import MetricWrapperBase
+from prometheus_client.utils import INF
+from typing import Dict, Optional
+import os
+from prometheus_client import CollectorRegistry
+from prometheus_client.multiprocess import MultiProcessCollector
+
+
+METRICS_PARAMS = {
+    "requests": (
+        Counter,
+        {
+            "name": "http_requests_total",
+            "documentation": "Total count of requests by method, path, and status code.",
+            "labelnames": ["method", "path_template", "status_code"],
+        },
+    ),
+    "requests_duration": (
```

Comment:
I think we can get to histograms later, can we just use a gauge to measure time?

Reply:
Ack